docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -444,7 +444,7 @@ For example, the `ToolbarComponent` has been replaced by a `Toolbar` component s
 
 ### Toolbar (`ToolbarComponent`)
 
-- The `ToolbarComponent` has been replaced by a `Toolbar` component slot:
+- ✅ The `ToolbarComponent` has been replaced by a `Toolbar` component slot:
```

Comment:
Maybe we should change these bullet points into sub-sections to add visibility of partial codemod coverage via side menu?
Or maybe come up with an intermediate icon for such section heading denoting that they are partly covered by codemods? 🤔

Reply:
It's inspired from the data grid. I prefer to have the same emoji, such that you know green square means "I can skip up to the next subsection/bullet point"

Reply:
I was referring to adding a different emoji to `### Toolbar (`ToolbarComponent`)` heading if at least some of it's subsections are covered, like in this case. Just for easier "scanning" when looking at side navigation.
Another mentioned option—change the bullet points into subheadings so that they would appear as submenu items in side nav and become more discoverable. 😃

Reply:
I don't get what is the interest of being "discoverable". 

It's more important to do not miss a breaking change not handled by codemod than seeing what is done by codemod

For me the current state says
> `Toolbar` is not handled. Check carefully what remains to be done.

Reply:
Well, fair point. Maybe it's best not to excite users about having as much green checkmarks as possible, but to let them attentively follow the guide. 👌